subprojects/core/src/main/java/org/gradle/internal/classpath/Instrumented.java

```diff
@@ -81,54 +82,36 @@ public static void discardListener() {
         setListener(NO_OP);
     }
 
+    private static final Map<String, CallInterceptor> GROOVY_CALLSITE_INTERCEPTORS = new HashMap<>();
+
+    static {
```

Comment:
💄 I'd use a static factory method here, `GROOVY_CALLSITE_INTERCEPTORS = buildCallSiteInterceptors();`

That way the factory method can use use a simpler name for the map as it builds it.

Reply:
Thank you for the suggestion! I solved it in another way.